CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Ruby 2.4.1 testing
+
+### Breaking Changes
```

Comment:
This is not technically a breaking change since we previously did not support ruby < 2 which is what I see as the minimum version required here: https://rubygems.org/gems/rest-client/versions/2.0.2 am I missing something? Assuming not we can have it under `### Changes`